src/routes/auth.js

```
@@ -32,4 +36,29 @@ router.patch(
   asyncWrapper(authController.updatePassword)
 )
 
+router.get('/confirm-email', async (req, res) => {
+  const token = req.query.token;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)